description.md

```diff
@@ -1,37 +1,44 @@
-AppMap enhances the developer experience by analyzing runtime code behavior before you commit changes and go to production.
+AppMap is a runtime analysis and visualization platform. It analyzes your application's runtime code behavior before your changes go to production, and allows you to understand complex code interactions.
 
-AppMap records detailed traces of how your application runs and analyzes it based on frameworks and techniques including web services, view templates, HTTP client requests, caching, authentication, and SQL.
+## Runtime behavior visualization
+After running your application with the AppMap library, the runtime recordings are instantly visualized in the IDE. These visualizations include:
```

Comment:
Maybe one screen shot for each? This really shows what the extension is.

Reply:
Yes, for sure, and those are already in the marketplace listing (it has a screenshots section that you upload images to). I'd like to use that for now instead of embedded images because the plugin installation widget in JetBrains has native support for showing screenshots, so it's really the "blessed" way of adding graphics.